tests/client/amo/helpers.js

```diff
@@ -50,8 +50,9 @@ export const fakeReview = {
 /*
  * Redux store state for when a user has signed in.
  */
-export const signedInApiState = {
+export const signedInApiState = (overrides = {}) => ({
```

Comment:
What happened here? Did some test accidentally mutate this object? If that was the only problem, you could probably fix it like this:
````js
export const signedInApiState = Object.freeze({
  lang: 'en-US',
  clientApp: 'firefox',
  token: 'secret-token',
});
````